assets/html/index.html

```diff
@@ -71,13 +94,13 @@ <h2>Serif font</h2>
 			alt="Cat">
 
 		<p>Avatar:</p>
-		<img src="../images/blank.png" data-src="https://0.gravatar.com/avatar/f3c5bdea23e1cce8cd23ecd0b2d773ae?s=100&amp;d=http%3A%2F%2F1.gravatar.com%2Favatar%2Fad516503a11cd5ca435acc9bb6523536%3Fs%3D25&amp;r=PG" alt="Gravatar" />
+		<img src="../images/blank.png" data-src="https://0.gravatar.com/avatar/f3c5bdea23e1cce8cd23ecd0b2d773ae?s=100&amp;d=http%3A%2F%2F1.gravatar.com%2Favatar%2Fad516503a11cd5ca435acc9bb6523536%3Fs%3D25&amp;r=PG" alt="Gravatar" loading="lazy" class="lazyload" />
```

Comment:
Do we even need the `class="lazyload"` here (and in other `img` tags)?

I don't see this class being used anywhere in the repo. I noticed, though, a `lazy` class being used in the [global stylesheet](https://github.com/xwp/xwpwa/pull/5/files#diff-f5450050229ee37e25a7192afd9d6960L191-L198).

Reply:
The `class="lazyload"` is used to target the image and change the `data-src` by src.
https://github.com/xwp/xwpwa/blob/fa40d21e3e034439efee02db5a7a35c6c0813a5a/assets/src/js/LazyLoad.js#L8

Indeed there is a fallback for the browsers how don't support the `loading="lazy"` attribute.
And I don't want to apply lazy loading for every images, an hero image in the viewport doesn't need that for example.

I commented the old CSS code to do a transition with opacity, her role was only cosmetic